src/packages/gaia-service/index.ts

```diff
@@ -15,15 +15,15 @@ export class GaiaService {
         this.gaiaWriteUrl = gaiaWriteUrl;
     }
 
-    public uploadContentToGaiaHub = async (filename: UPLOADABLE_JSON_FILES, privKey: string, content: any, prefix?: string, type = "application/json"): Promise<string> => {
+    public uploadContentToGaiaHub = async (filename: UPLOADABLE_JSON_FILES, privKey: string, content: any, prefix: string, type = "application/json"): Promise<string> => {
         const sanitizedPrivKey = sanitizePrivKey(privKey);
         const hubURL = this.gaiaWriteUrl;
         const hubConfig = await blockstack.connectToGaiaHub(hubURL, sanitizedPrivKey);
         const tokenFile = this._generateTokenFileForContent(sanitizedPrivKey, content);
         const contentToUpload: string = JSON.stringify(tokenFile);
         let finalURL: string;
         try {
-            finalURL = await blockstack.uploadToGaiaHub((prefix ? `${prefix}_` : "") + filename, contentToUpload, hubConfig, type);
+            finalURL = await blockstack.uploadToGaiaHub(`${prefix}_` + filename, contentToUpload, hubConfig, type);
```

Comment:
As discussed, lets move the prefix logic to BlockstackService class, and keep GaiaService as generic as possible.